### PR TITLE
Switched to stream-io-api.com for API and Analytics

### PR DESCRIPTION
**Pull request by dwightgunning**

### CODE REVIEW

File: lib/GetStream/Stream/Client.php [hunk 1]
```diff
@@ -152,7 +152,7 @@ public function getBaseUrl()
             if ($this->location) {
                 $subdomain = "{$this->location}-api";
                 if ($this->location == 'qa') {
-                    $api_endpoint = 'getstream.io/api';
+                    $api_endpoint = 'stream-io-api.com/api';
                     $this->setProtocol('http');
```
Comment (tbarbugli) [inline lib/GetStream/Stream/Client.php]: please remove this, we now have HTTPS on QA too

Comment (dwightgunning) [reply lib/GetStream/Stream/Client.php]: Will do.

**Comment by dwightgunning** [comment]
Fixed https QA.

Included an extra commit to support STREAM_BASE_URL environment variable.

**Comment by dwightgunning** [comment]
Discussed in person with Tommaso. We're good to merge.

**Pull request merged**